.github/workflows/docker_publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Docker Hub publish
+
+on:
+  push:
+    branches:
+      - "develop"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Login to Docker Hub
```

Comment:
As you may know, we have to move to github registry :/ 
Look at this one https://github.com/IGNF/apicarto/blob/master/.github/workflows/docker-publish.yml it seems easy there. ^^

Reply:
updated to use github container registry